src/main/java/io/appform/dropwizard/discovery/bundle/ServiceDiscoveryBundle.java

```diff
@@ -195,12 +195,7 @@ protected int getPort(T configuration) {
     }
 
     protected String getHost() throws UnknownHostException {
-        val host = (Strings.isNullOrEmpty(serviceDiscoveryConfiguration.getPublishedHost())
-                || serviceDiscoveryConfiguration.getPublishedHost()
-                .equals(Constants.DEFAULT_HOST))
-                   ? InetAddress.getLocalHost()
-                           .getCanonicalHostName()
-                   : serviceDiscoveryConfiguration.getPublishedHost();
+        val host = serviceDiscoveryConfiguration.getPublishedHost();
```

Comment:
This doesn't look right. Mostly when the software is not running in a container, providing hostname should not be necessary